src/Dockerfile.fedora

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/{{ spec.latest_fedora }}/s2i-core:latest
+FROM quay.io/fedora/s2i-core:35
```

Comment:
What about using tag "latest" here? We would not have to rewrite the Dockerfiles everytime a new Fedora is released, if we use latest.

Reply:
Definitely not. Although `s2i-core container` has the latest tag, which is not good, we would like to tell the user, which Fedora version is used. `Latest` tag means, nowadays `36`, after a Fedora release.